test_requirements.txt

```diff
@@ -2,7 +2,6 @@
 colour-runner==0.0.4
 coverage==3.7.1
 dj-database-url==0.3.0
-django==1.7.4
```

Comment:
Updated in https://github.com/incuna/django-pgcrypto-fields/commit/1d7f055106c3bf485aaad93a6b4cb0b09f54bb3c